pyproject.toml

```
@@ -50,7 +50,7 @@ dependencies = [
 # Recommended run-time dependencies to enable a lot of functionality within Astropy.
 recommended = [
     "scipy>=1.8",
-    "matplotlib>=3.3,!=3.4.0,!=3.5.2",
+    "matplotlib>=3.5.0,!=3.5.2",
```

Here's what I get for coding on a train

```suggestion
    "matplotlib>=3.6.0",
```